### DISCUSSION TITLE
[Question]:  Getting started - how to detect household events?

### DISCUSSION THREAD

**Discussion by alfista2600** [blakeblackshear/frigate#10387] [Ask A Question]
### What is your question:

Hi! I have some cameras set up that are not for security, but to keep an eye on things in the house - like when the dog's water has been changed, when he was fed and when he has emptied the bowl. Is there a good article somewhere on how to set this up with good detection parameters so I don't record the dog eating for 10 minutes? He eats slowly, and not timely when food is put out, so want to get some alerts when not around. I poked around but didn't see one. Thanks for tips, and apologies if I missed something.

**Comment by nwish** [comment]
You can set up zones and then use Home Assistant to detect when your zones are occupied by a certain object, or cleared for a certain amount of time.  I’m not sure exactly what you want alerts for, but that sounds like something that could work.

**Reply by alfista2600** [reply]
Thanks - yes, that is along the line of what I'm thinking (I use OpenHAB, however). I'm mostly just trying to deploy settings for Frigate that will reduce the amount of processing I need to do in my automation rule (e.g. Dog gets piece of food, leaves with it, comes back to get another) I don't want to trigger multiple events, or record a long video... looking for a best practice for this if there is one.

**Reply by alfista2600** [reply]
also 
bowl -> bowl with food 
bowl with food -> bowl
would be cool if I could detect

I guess I could do:
bowl enters zone
dog enters zone
as a slight proxy to eating... but he does a lot of sniffing and evaluating... and might not eat for many hours

I've signed up for frigate+ so maybe I can train a model for a full bowl? still learning about all this...

**Reply by nwish** [reply]
Right now, Frigate+ doesn’t yet support training custom models outside the list of base objects, but Frigate in general does support custom models.  

I feel like there are too many variables to cover in order to accomplish your goal, but I could be wrong!  

Hey @NickM-27, is this something you’d think Swatch could partially handle?

**Reply by NickM-27** [reply]
maybe but not sure it would be accurate enough depending on the lighting changes where the bowl would be . Haven't had any time to put in to that project in a while either
